datasets/leidingeninfrastructuur/leidingeninfrastructuur.json

```diff
@@ -0,0 +1,238 @@
+{
+  "type": "dataset",
+  "id": "leidingeninfrastructuur",
+  "title": "Kabels en leidingen ondergrond",
+  "status": "beschikbaar",
+  "description": "Overzicht leidingen infrastructuur op basis van klic meldingen",
+  "version": "0.0.1",
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "liggingLijnTotaal",
+      "type": "table",
+      "provenance": "leidingeninfrastructuur_ligging_lijn_totaal",
+      "title": "leidingeninfrastructuur lijn",
+      "description": "Overzicht lijnen infrastructuur op basis van klic meldingen",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["schema", "id"],
+        "mainGeometry": "geometrie",
+        "display": "id",
+        "properties": {
+          "id": {
+            "type": "integer",
+            "description": "Unieke id van het object"
+          },
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "thema": {
+            "type": "string",
+            "description": "Thema of discipline waar een leiding of leidingelement toe behoort"
+          },
+          "class": {
+            "type": "string",
+            "description": "Objectklasse of type kabel/leiding"
+          },
+          "beheerder": {
+            "type": "string",
+            "description": "Degene die als natuurlijk persoon handelende in de uitoefening van een beroep of bedrijf dan wel als rechtspersoon een net beheert."
+          },
+          "diamMm": {
+            "type": "number",
+            "description": "Diameter buis of leiding"
+          },
+          "diepte": {
+            "type": "number",
+            "description": "Diepte t.o.v. maaiveld"
+          },
+          "materiaal": {
+            "type": "string",
+            "description": "Materiaal van de kabel of leiding"
+          },
+          "voltage": {
+            "type": "string",
+            "description": "Het operatieve en nominale voltage van de elektriciteitskabel (indien van toepassing)"
+          },
+          "status": {
+            "type": "string",
+            "description": "De status van het object"
+          },
+          "buurtCode": {
+            "type": "string",
+            "provenance": "buurt_code",
```

Comment:
Als de kolomnaam hetzelfde is als de bron, dan hoef je geen provenance te gebruiken. In dit geval zal er ook een error komen omdat er dan een rename wordt gedaan op een kolom die de naam al heeft "already exists"